node/chainSimulator/components/coreComponents.go

```diff
@@ -90,7 +90,7 @@ type ArgsCoreComponentsHolder struct {
 }
 
 // CreateCoreComponentsHolder will create a new instance of factory.CoreComponentsHolder
-func CreateCoreComponentsHolder(args ArgsCoreComponentsHolder) (factory.CoreComponentsHolder, error) {
+func CreateCoreComponentsHolder(args ArgsCoreComponentsHolder) (factory.CoreComponentsHandler, error) {
```

Comment:
Please rename to `CreateCoreComponents` and remove the inner closeHandler declaration & usage

Reply:
renamed